### ISSUE TITLE
Broken links in documentation

### ISSUE THREAD

**Issue by Ghepardo** [thlee/SNPhylo#44]
Several of the URLs in the documentation (at least in the installation instructions and on the GitHub page itself) are broken.  I have not checked them exhaustively, but at least the following are broken:

http://chibba.pgml.uga.edu/snphylo/snphylo.tar.gz
http://chibba.pgml.uga.edu/snphylo/soybean.hapmap.gz

**Comment by Yyeserin** [comment]
Yes! A manual will be veryhelpful. Is there a way to add it to the github page?
